dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/DotNet/IVectorDb.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
+
+namespace Qdrant.DotNet;
+
+public interface IVectorDb
```

Comment:
We should certainly evaluate whether these interfaces are needed here in the qdrant connector, or if they belong in the Kernel as part of IMemoryStore or IEmbeddingIndex

Reply:
I would expect these interfaces to be part of the kernel.  There will be other vector dbs we will need to support.